storage/storage_helpers.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Maybe this should go under `storage/sql` or something - explicitly helpers for golang SQL  compatible implementations?